db/seeds.rb

```diff
@@ -1,6 +1,8 @@
 # users ------------------------------------------------------------------------
 users_params = {
-  gurl: { username: "gurl", email: "gurl@example.com", password: "gurl", password_confirmation: "gurl", bio: "Just a midwest gurl." }
+  gurl: { username: "gurl", email: "gurl@example.com", password: "gurl", password_confirmation: "gurl", bio: "Just a midwest gurl." },
```

Comment:
Had to add a comma, which is why this is highlighted. (Same with everything else below. I did not delete anything.)